src/transformers/modeling_tf_t5.py

```diff
@@ -1069,7 +1067,7 @@ def call(
 
 
 @add_start_docstrings("""T5 Model with a `language modeling` head on top. """, T5_START_DOCSTRING)
-class TFT5ForConditionalGeneration(TFT5PreTrainedModel, TFCausalLanguageModelingLoss):
+class TFT5ForConditionalGeneration(TFT5PreTrainedModel, TFCausalLanguageModeling ):
```

Comment:
I think this should stay as `TFCausalLanguageModelingLoss `

Reply:
Sorry I forgot to delete this quick test

Reply:
This is true I unintentionally change it